scipy/sparse/linalg/eigen/lobpcg/lobpcg.py

```diff
@@ -31,8 +31,8 @@ def save(ar, fileName):
     savetxt(fileName, ar, precision=8)
 
 
-def _assert_symmetric(M, rtol=1e-5, atol=1e-8):
-    assert_allclose(M.T.conj(), M, rtol=rtol, atol=atol)
+# def _assert_symmetric(M, rtol=1e-5, atol=1e-8):
+#     assert_allclose(M.T.conj(), M, rtol=rtol, atol=atol)
```

Comment:
Just to be sure--the fact that the math here is rather challenging justifies retaining an unused function as a comment? I see the mathematician providing this solution in the linked issue has suggested to comment out.

Reply:
This is a work-in-progress solution -  the lines will be either removed or a warning will be issued. @lobpcg wants to collect more data on why np.float32 matrices sometimes fail, see the linked issue comment.

Reply:
You can add "WIP:" to the title to make it a bit more obvious

Reply:
Done, thanks for the tip!